### PR TITLE
Feat: Home-Auth pages are added

### PR DESCRIPTION
**Pull request by ramalingaa**
Landing page is added
Authentication(login-signup-passwordreset) pages are added

### CODE REVIEW

File: src/frontend/context/video-context.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                const response = await axios.get("/api/videos")
+                dispatch({type:"SET_VIDEO_DATA", payload:response.data.videos})
+                
+            }catch(e) {
+                console.log(e)
+            }
+        })()
+    },[])
+
+    useEffect(() =>{
```
Comment (Rohanmond) [inline src/frontend/context/video-context.js:27]: you can use one useEffect as it is for the same reason

**Review by Rohanmond — APPROVED** [review]
code looks so clean

**Review by Shah-Arsalan — APPROVED** [review]
Very well written code ! You can merge

**Review by Logan1x — APPROVED** [review]
The extra useeffect can be refactored, Everything else looks good, You can proceed.

**Pull request merged**